graphql-dgs/src/main/kotlin/com/netflix/graphql/dgs/internal/DgsSchemaProvider.kt

```diff
@@ -445,31 +449,30 @@ class DgsSchemaProvider(
                 return@async method.kotlinFunction!!.callSuspend(dgsComponent, *args.toTypedArray())
             }
 
-            launch?.asCompletableFuture()
+            launch.asCompletableFuture()
```

Comment:
launch no longer nullable?

Reply:
This might be left over from some refactoring when it was nullable? But yeah it's not nullable on master, so the `?.` isn't necessary.